shared/chat/conversation/index.native.js

```diff
@@ -30,30 +32,47 @@ const Conversation = (props: Props) => (
           Couldn't load all chat messages due to network connectivity. Retrying...
         </Text>
       </Box>}
-    <Header
-      sidePanelOpen={props.sidePanelOpen}
-      onToggleSidePanel={props.onToggleSidePanel}
-      onBack={props.onBack}
-    />
-    <List
-      focusInputCounter={props.focusInputCounter}
-      listScrollDownCounter={props.listScrollDownCounter}
-      onEditLastMessage={props.onEditLastMessage}
-      onScrollDown={props.onScrollDown}
-      onFocusInput={props.onFocusInput}
-      editLastMessageCounter={props.editLastMessageCounter}
-    />
-    <Banner />
-    {props.showLoader && <LoadingLine />}
-    {props.finalizeInfo
-      ? <OldProfileResetNotice />
-      : <Input
-          focusInputCounter={props.focusInputCounter}
-          onEditLastMessage={props.onEditLastMessage}
-          onScrollDown={props.onScrollDown}
+    {props.inSearch
```

Comment:
maybe we should break these out into their own constants or something
const Header = props.inSearch ? ...
then just {Header} here